src/app/about/About.module.css

```diff
@@ -0,0 +1,30 @@
+.about {
+    /* styles */
+    text-align: center;
+}
+
+.about__image_container {
+    display: flex;
+    justify-content: center;
+    margin-bottom: 2rem;
```

Comment:
[Please Change]: The `margin-bottom` pushes down the edge of page, which looks particularly odd in combination with the background gradient. No need to have this since it's end of page.

![image](https://github.com/ufssd/ufssd-website/assets/35618116/48f61ac6-952a-4b57-b2dc-b907d29cb3fb)